draft-ietf-wish-whep.md

```diff
@@ -633,6 +633,25 @@ data: /DA8AAAAAAAAAP///wb+06ACpQAmAiRDVUVJAACcHX//AACky4AMEERJU0NZTVdGMDQ1MjAwME
 ~~~~~
 {: title="scte35 example event"}
 
+### Events over WebRTC datachannel
+
+In some architectures sending events over a long polled HTTP request from a server that isn't the media server may be costly, such servers may support events over the webRTC datachannel.
+The protocol is identical to the above, with the following exceptions:
+
+- If both the SDP offer and the SDP answer contain an application section this indicates that Datachannel events are supported.
+~~~~~
+m=application 9 UDP/DTLS/SCTP webrtc-datachannel
+~~~~~
+- If the client then opens a reliable datachannel with a label that matches the URI of the sse Location: the server MUST send events as messages over this datachannel in the format described above.
```

Comment:
should we open a datachannel based on the event name and not on the sse location?

Reply:
do we need to specify the reliable/ordered/negotiated flags of the datachannel?

Reply:
I think to remain true to the existing SSE semantics we probably want to say reliable+ordered+!negotiated but others may have different views.

The location as label feels simpler and more flexible. Using event names as labels requires multiple data channels to be opened, then the ordering of the event arrival is non-deterministic - which I _think_ isn't the case with SSE.